packages/core/src/fetchers/index.ts

```diff
@@ -2,3 +2,6 @@ export * from './fetchBySlug';
 export * from './fetchById';
 export * from './fetchAllEntities';
 export * from './createExperience';
+// Exposing the underlying fetcher methods to allow caching of API responses
+export * from './fetchReferencedEntities';
+export * from './fetchExperienceEntry';
```

Comment:
You will need to re-export these from `packages/experience-builder-sdk/src/index.ts`, otherwise they are not really exposed to the end-user of the SDK.